tests/continuous/test_consistent.py

```diff
@@ -0,0 +1,239 @@
+    data=[[nan, ALPHA * 4.0, nan], [0.0, ALPHA * 100, (1 - ALPHA) * 4]],
+    dims=["date", "station"],
+    coords=dict(date=["1", "2"], station=[100, 101, 102]),
+)
+
+EXP_EXPECTILE_SCORE2 = xr.DataArray(
+    data=[0.0, ALPHA * (4 + 100) / 2, (1 - ALPHA) * 4],
+    dims=["station"],
+    coords=dict(station=[100, 101, 102]),
+)
+EXP_EXPECTILE_SCORE3 = xr.DataArray((ALPHA * 4.0 + ALPHA * 100 + (1 - ALPHA) * 4) / 4)
+
+TUNING_PARAM = 2.0
+
+EXP_HUBER_SCORE1 = xr.DataArray(
+    data=[[nan, 2.0, nan], [0.0, 18.0, 2.0]],
+    dims=["date", "station"],
+    coords=dict(date=["1", "2"], station=[100, 101, 102]),
+)
+
+EXP_HUBER_SCORE2 = xr.DataArray(
+    data=[2.0, 20 / 3],
+    dims=["date"],
+    coords=dict(date=["1", "2"]),
+)
+
+EXP_HUBER_SCORE3 = xr.DataArray(
+    data=22 / 4,
+)
+
+EXP_QUANTILE_SCORE1 = xr.DataArray(
+    data=[[nan, ALPHA * 2.0, nan], [0.0, ALPHA * 10, (1 - ALPHA) * 2]],
+    dims=["date", "station"],
+    coords=dict(date=["1", "2"], station=[100, 101, 102]),
+)
+
+EXP_QUANTILE_SCORE2 = xr.DataArray(
+    data=[ALPHA * 2.0, (ALPHA * 10 + (1 - ALPHA) * 2) / 3],
+    dims=["date"],
+    coords=dict(date=["1", "2"]),
+)
+
+EXP_QUANTILE_SCORE3 = xr.DataArray((ALPHA * 2.0 + ALPHA * 10 + (1 - ALPHA) * 2) / 4)
+
+
+def squared_loss(x):
+    """squared loss function"""
+    return x**2
+
+
+def squared_loss_prime(x):
+    """derivative of `squared_loss`"""
+    return 2 * x
+
+
+def simple_linear(x):
+    """simple increasing function"""
+    return x
+
+
+@pytest.mark.parametrize(
+    ("preserve_dims", "reduce_dims", "weights", "expected"),
+    [
+        (["date", "station"], None, None, EXP_EXPECTILE_SCORE1),
+        (["date", "station"], None, WEIGHTS, 2 * EXP_EXPECTILE_SCORE1),
+        (["station"], None, None, EXP_EXPECTILE_SCORE2),
+        (None, ["date"], None, EXP_EXPECTILE_SCORE2),
+        (None, None, None, EXP_EXPECTILE_SCORE3),
+    ],
+)
+def test_consistent_expectile_score(preserve_dims, reduce_dims, weights, expected):
+    """Tests that `consistent_expectile_score` gives results as expected."""
+    result = consistent_expectile_score(
+        DA_FCST,
+        DA_OBS,
+        alpha=ALPHA,
+        phi=squared_loss,
+        phi_prime=squared_loss_prime,
+        preserve_dims=preserve_dims,
+        reduce_dims=reduce_dims,
+        weights=weights,
+    )
+    assert_allclose(result, expected)
+
+
+def test_expectile_score_with_dask():
+    """
+    Tests that the consistent expectile scores work with Dask
+    """
+    result = consistent_expectile_score(
+        DA_FCST.chunk(),
+        DA_OBS.chunk(),
+        alpha=ALPHA,
+        phi=squared_loss,
+        phi_prime=squared_loss_prime,
+        preserve_dims="all",
+    )
+    assert isinstance(result.data, dask.array.Array)
+    result = result.compute()
+    assert isinstance(result.data, np.ndarray)
+    assert_allclose(result, EXP_EXPECTILE_SCORE1)
+
+
+@pytest.mark.parametrize("alpha", [1.0, 0.0])
+def test_check_alpha(alpha):
```

Comment:
I don't see much point in using parametrize for just one case. Best to rename the test to test_check_alpha_raises_on_invalid, and just putting a number into it.

Reply:
No, just leave it, I think I just didn't notice it was in fact two cases when reading through, thanks.